addons/web/static/src/webclient/user_menu/user_menu_items.js

```diff
@@ -48,6 +47,7 @@ function shortCutsItem(env) {
         id: "shortcuts",
         hide: env.isSmall,
         description: env._t("Shortcuts"),
+        hotkey: `${isMacOS() ? "CMD" : "CTRL"}+K`,
```

Comment:
as we use t-out to render the description, there's no need to add a new mechanism for the hotkey: you could set a description which contains html, to display the hotkey in bold, right here